src/style.css

```diff
@@ -0,0 +1,162 @@
+/* #region reset */
+body {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+}
+/* #endregion */
+
+/* #region card */
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+
+  --min-element-margin: 16px;
+  --size-text-s: 10px;
+  --size-text-m: 12px;
+  --size-text-l: 14px;
+  --size-text-xl: 16px;
+  --color-gray: #616070;
+  --color-dark-blue: #060b35;
+  --color-blue: #00acdc;
+}
+
+.card__img-postion {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  font-family: Roboto, sans-serif;
+  font-size: var(--size-text-m);
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--color-dark-blue);
+}
+
+.card__id-text {
+  font-family: Roboto, sans-serif;
```

Comment:
Move all `font-family` rules to the body and don't repeat it for every block. It will be inherited from the body.